File: app/src/main/java/ru/programminglearning/com/hamsterProg/Registration/Welcome/WelcomeActivity.java
package ru.programminglearning.com.hamsterProg.Registration.Welcome;


import android.support.v4.app.Fragment;

import ru.programminglearning.com.hamsterProg.Registration.RegistrationActivityMain;

public class WelcomeActivity extends RegistrationActivityMain {

    @Override
    protected Fragment getFragment() {
        return WelcomeFragment.newInstance();
    }
}

File: app/src/main/java/ru/programminglearning/com/hamsterProg/Registration/Welcome/WelcomeActivity.kt
package ru.programminglearning.com.hamsterProg.Registration.Welcome

import android.support.v4.app.Fragment
import ru.programminglearning.com.hamsterProg.Registration.RegistrationActivityMain

class WelcomeActivity : RegistrationActivityMain() {
    override val fragment: Fragment?
        get() =  WelcomeFragment.newInstance()
}